lib/fixer.js

```diff
@@ -83,7 +83,12 @@ var fixer = module.exports = {
     if (!data.bin) return;
     if (typeof data.bin === "string") {
       var b = {}
-      b[data.name] = data.bin
+      var match
+      if (match = data.name.match(/^@[^/]+[/](.*)$/)) {
```

Comment:
request: Can we do package name parsing be its own standalone module? We can do package name validation in that independent module, too.